src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Index.cshtml

```diff
@@ -27,7 +27,11 @@
             </details>
             <p>You can browse all opportunities available.</p>
             <p>Some businesses have chosen not to share their organisation's name publicly so these opportunities just show a reference number. If your application is successful, we'll let you know who the employer willing to fund your apprenticeship training is.</p>
-            <h2 class="govuk-heading-m">@Model.Opportunities.Count() opportunities</h2>
+            <h2 class="govuk-heading-m">@Model.Opportunities.Count() 
+            @if (Model.Opportunities.Count() == 1)
```

Comment:
@(Model.Opportunities.Count() == 1 ? "opportunity" : "opportunities") is cleaner